### PR TITLE
Implement new comments

### LINKED ISSUE #6: New Comments UI

### LINKED ISSUE #10: Pinned comments
Needs #6 - could you handle this one if i do APIs @EXtremeExploit?

### PR DESCRIPTION
**Pull request by Tanza3D**
dropdown doesn't work just yet so draft for the time being..

### CODE REVIEW

**Comment by Tanza3D** [comment]
todo:
- [ ] restriction icon
- [x] pinned icon
- [x] group badges
- [x] dropdowns

**Comment by Tanza3D** [comment]
really need actual testing on this one!!

**Comment by Tanza3D** [comment]
should be all done now, im not adding the restriction badge for the time being, don't see it being too important

File: global/css/comments.css [hunk 1]
```diff
@@ -1,401 +1,293 @@
...
+    font-size: 12px;
+    margin-right: 6px;
 }
 
-.comments__comment-vote-voted {
-    background: #fff;
+.comments__post-button {
+    padding: 0px 25px;
+    height: 36px;
+    display: flex;
```
Comment (Coppertine) [inline global/css/comments.css:251]:
```suggestion
    display: flex;
    gap: 5px;
```
image
Adds padding to send icon and text for readability.

Comment (Tanza3D) [reply global/css/comments.css:251]: oh, good idea, forgot this

File: global/css/comments.css [hunk 2]
```diff
@@ -1,401 +1,293 @@
...
+    align-items: center;
     color: rgb(var(--accentdark));
+    background-color: white;
+    border-radius: 0px 0px 10px 0px;
+    margin-left: 4px;
```
Comment (Coppertine) [inline global/css/comments.css:257]:
```suggestion
    margin-left: 4px;
    cursor: pointer;
```
WHY NO CURSOR?

File: index.php [hunk 3]
```diff
@@ -127,19 +131,91 @@
...
+                                                </div>
+                                            </div>
+                                            <div class="comments__comment-content-text">
+                                                <p>look at me im an awesome comment</p>
+                                            </div>
+                                        </div>
+                                        <div class="comments__comment-infobar">
+                                            <div class="comments__comment-infobar-info">
+                                                <i class="fas fa-calendar-alt"></i>
+                                                <p>posted <strong>5 days ago</strong></p>
```
Comment (Coppertine) [inline index.php:177]: Should add a tooltip for localised time of comment.

File: global/css/comments.css [hunk 4]
```diff
@@ -1,401 +1,293 @@
...
+    margin-left: 6px;
+    height: 21px;
+    width: 35px;
+    background: linear-gradient(180deg, rgba(0, 0, 0, 0.25) 0%, rgba(0, 0, 0, 0.1125) 100%);
+box-shadow: inset 2px 4px 6px rgba(0, 0, 0, 0.1);
+border-radius: 99px;
+display: flex;
+align-items: center;
+justify-content: center;
+font-size: 14px;
```
Comment (Coppertine) [inline global/css/comments.css:191]:
```suggestion
font-size: 14px;
cursor: pointer;
```
this is what I ment by cursor...

File: global/css/comments.css [hunk 5]
```diff
@@ -1,401 +1,295 @@
...
-    font-size: 16px;
-    white-space: pre-line;
+.comments__comment-content img {
+    max-width: 10%;
 }
 
-.comments__comment-vote, .comments__comment-reply, .comments__comment-delete, .comments__comment-report {
-    width: 40px;
-    background: linear-gradient(#fff1, #0001), linear-gradient(rgba(var(--accentdark), 1), rgba(var(--accentdark), 1));
+.comments__comment-votes {
```
Comment (Coppertine) [inline global/css/comments.css:78]: Consider a hover and selected event for upvoting a comment.

File: global/css/comments.css [hunk 6]
```diff
@@ -1,401 +1,295 @@
...
+    padding: 6px;
+    min-width: 40px;
+    text-align: center;
     display: flex;
-    justify-content: center;
-    align-content: center;
-    justify-items: center;
     align-items: center;
+    justify-content: center;
+    margin-top: 8px;
```
Comment (Coppertine) [inline global/css/comments.css:87]:
```suggestion
    margin-top: 8px;
    cursor: pointer;
```

File: index.php [hunk 7]
```diff
@@ -127,19 +131,91 @@
...
+                                    <div class="comments__comment-left">
+                                        <img src="https://a.ppy.sh/1" class="comments__pfp">
+                                        <div class="comments__comment-votes">
+                                            +22
+                                        </div>
+                                    </div>
+                                    <div class="comments__comment-right">
+                                        <div class="comments__comment-content">
+                                            <div class="comments__comment-content-username">
+                                                <p class="comments__username">Username</p>
```
Comment (Coppertine) [inline index.php:165]: Consider as a link to the osekai-profile of said user.

**Review by Coppertine — CHANGES_REQUESTED** [review]
Try to find places where you can click, and set them to `cursor: pointer;` when you can.
please....

File: index.php (hunk 3 above)
Comment (Tanza3D) [reply index.php:177]: image
there is

File: global/css/comments.css (hunk 5 above)
Comment (Tanza3D) [reply global/css/comments.css:78]: have done

File: index.php (hunk 7 above)
Comment (Tanza3D) [reply index.php:165]: have done (please stop looking at the html on index.php it's not representative of what's on the page, it was just to test css...)

**Comment by Tanza3D** [comment]
fixed light mode

**Review by Coppertine — APPROVED** [review]
lgtm

**Pull request merged**